test/library/draft/DistributedMap/v3/staticMemManager.chpl

```diff
@@ -0,0 +1,11 @@
+use DistributedMap;
+
+var dm = new distributedMap(int, int);
+var staticManager = new staticRefsManager(dm);
+
+forall i in 0..100 with (ref staticManager) do
+    manage staticManager as staticMap do // the underlying map is notified that it can't rehash right now
+      staticMap[i] += 1;
+
+// <- here: the map is notified that it can rehash
```

Comment:
I think you want to ensure that this test generates correct output, it could "work" but have done something wrong, right?

Reply:
Yeah that's a good point. The context manager stuff isn't finished yet so I didn't finish writing this test, but I was planning on working on it next sprint. I'll make a note of it for now though.